ui/widgets.go

```diff
@@ -34,21 +39,28 @@ type combined struct {
 
 type outputs struct {
 	icons struct {
-		add, check, cancel, sync decredmaterial.IconButton
+		add, check, cancel, sync, info, more, copy decredmaterial.IconButton
 	}
+
 	spendingPassword, matchSpending                                            decredmaterial.Editor
-	toOverview, toWallets, toRestoreWallet, toTransactions, toSend, toSettings decredmaterial.IconButton
-	//toReceive                                                 decredmaterial.IconButton
-	createDiag, cancelDiag decredmaterial.IconButton
+	toOverview, toWallets, toTransactions, toRestoreWallet, toSend, toSettings decredmaterial.IconButton
+	toReceive                                                                  decredmaterial.IconButton
+	createDiag, cancelDiag                                                     decredmaterial.IconButton
 
-	createWallet, restoreDiag, restoreWallet, deleteWallet, deleteDiag decredmaterial.Button
-	sync                                                               decredmaterial.IconButton
+	createWallet, restoreDiag, restoreWallet, deleteWallet, deleteDiag, gotItDiag decredmaterial.Button
+	newAddress                                                                    decredmaterial.Button
+	sync, info, more, copy                                                        decredmaterial.IconButton
 
-	tabs                          []decredmaterial.TabItem
-	notImplemented, noWallet, err decredmaterial.Label
+	tabs                                                      []decredmaterial.TabItem
+	notImplemented, noWallet, receivePageTitle, pageInfo, err decredmaterial.Label
```

Comment:
`reveivePageTitle` -> `pageTitle`
I'm not even sure we need page titles.

Reply:
So are you recommending i remove it?

Reply:
Keep it if you want. It's really a design thing to have page titles but you'll probably have to have change the title for each page